cmd/harmony/main.go

```diff
@@ -429,7 +428,10 @@ func main() {
 	}
 
 	nodeconfig.SetPublicRPC(*publicRPC)
-	nodeconfig.SetVersion(fmt.Sprintf("Harmony (C) 2019. %v, version %v-%v (%v %v)", path.Base(os.Args[0]), version, commit, builtBy, builtAt))
+	nodeconfig.SetVersion(
```

Comment:
why you mix change of slash with the trivial version change?

Reply:
Just general practice of fix up easy tech debt along the way since my velocity is quick